test/libsolidity/semanticTests/libraries/library_staticcall_delegatecal.sol

```diff
@@ -0,0 +1,20 @@
+library Lib {
```

Comment:
Typo in test file name: `delegatecal` -> `delegatecall`.

Reply:
Thanks for pointing this out. 
Has been corrected.